Hookies/Game/Engine/GameEngine.swift

```diff
@@ -27,10 +27,12 @@ class GameEngine {
     private var userConnectionSystem: UserConnectionSystem?
     private var startSystem = StartSystem()
     private var endSystem: EndSystem?
+    private var botSystem: BotSystem?
 
     // MARK: - Entity
 
     private var currentPlayer: PlayerEntity?
+    private var localPlayers = [String: PlayerEntity]()
     private var otherPlayers = [String: PlayerEntity]()
```

Comment:
@jinyingtan We should remove the `otherPlayers` as it is not used anymore.

Reply:
I will remove it in another branch